tests/MongoDB.Bson.Tests/IO/BsonStreamAdapterTests.cs

```diff
@@ -54,7 +54,7 @@ public void BaseStream_should_throw_when_subject_is_disposed()
             action.ShouldThrow<ObjectDisposedException>().And.ObjectName.Should().Be("BsonStreamAdapter");
         }
 
-#if NET45
+#if NET452
```

Comment:
See comments in MongoDB.Bson.Tests.csproj for why this is NET452 instead of NET45.